nginx.conf

```diff
@@ -0,0 +1,50 @@
+user www-data;
```

Comment:
I'd move this file to a new folder. Or? What do you think?

Reply:
I think to, not sure that docker-compose.yml has to stay at the project's root directory too.
What do you think @uracreative ?

Reply:
@Borisbudini ?

Reply:
i love the mastodon approach of leaving the docker-compose.yml at the root of the project... but thats a decision for them to make.. :)

Reply:
Unless @Borisbudini has a very strong opinion on this, we trust you on that.

Reply:
Yeah, i think docker-compose.yml should be at the project's root aswell.